README.markdown

```diff
@@ -192,6 +192,38 @@ load-nvmrc() {
 }
 add-zsh-hook chpwd load-nvmrc
 ```
+##### Automatic version switching for `.bashrc` with fallback to `nvm use default`
+
+Put this into your `$HOME/.bashrc` to enable automatic version swithing whenever you enter a directory than contains an
+`.nvmrc` file with a string telling nvm which node to `use`:
+
+```bash
+cd() {
+  builtin cd "$@"
+  if [[ -f .nvmrc && -r .nvmrc ]]; then
+    nvm use --silent
+  elif [[ `nvm current` != `nvm version default` ]]; then
+    nvm use --silent default
```

Comment:
`nvm use --silent` will use nvmrc, and fall back to default for you.

In other words, this function could just be:
```bash
cd() {
  builtin cd "$@"
  nvm use --silent
}
```